composer.json

```diff
@@ -100,9 +100,10 @@
     "moderntribe/acf-menu-chooser": "^1.1",
     "moderntribe/square1-field-models": "^1.0",
     "moderntribe/tribe-acf-post-list-field": "^2.0",
-    "moderntribe/tribe-libs": "^3.4",
+    "moderntribe/tribe-libs": "dev-feature/block-middleware as 3.4.18",
```

Comment:
Note: Revert this once the companion PR is released: https://github.com/moderntribe/tribe-libs/pull/123